projects/ion/src/lib/switch/types.ts

```diff
@@ -0,0 +1,53 @@
+import { EventEmitter } from '@angular/core';
+
+/**
+import { IonSwitchComponent } from './switch.component';
```

Comment:
This import should really be commented here? 🤔

Reply:
🤡